tests/Application/src/Repository/PaymentMethodRepository.php

```diff
@@ -4,10 +4,11 @@
 
 namespace App\Repository;
 
-use CommerceWeavers\SyliusTpayPlugin\Repository\FindByChannelWithGatewayConfigTrait;
+use CommerceWeavers\SyliusTpayPlugin\Repository\PaymentMethodTrait;
+use CommerceWeavers\SyliusTpayPlugin\Repository\PaymentMethodRepositoryInterface;
 use Sylius\Bundle\CoreBundle\Doctrine\ORM\PaymentMethodRepository as BasePaymentMethodRepository;
 
 final class PaymentMethodRepository extends BasePaymentMethodRepository implements PaymentMethodRepositoryInterface
 {
-    use FindByChannelWithGatewayConfigTrait;
+    use PaymentMethodTrait;
```

Comment:
I think this is a simple and clean approach of naming repository traits. If we want to follow the previous convention, the trait name should be `FindByChannelAndGatewayNameWithGatewayConfigTrait` – that's long, and it's not clear in which repository the trait should be used.